Makefile

```diff
@@ -53,6 +53,15 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+GOLANGCI_LINT = $(GOBIN)/golangci-lint
+# golangci-lint version should be updated periodically
+# we keep it fixed to avoid it from unexpectedly failing on the project
+# in case of a version bump
+GOLANGCI_LINT_VER = v1.46.1
+TIMEOUT = 15
+Q = $(if $(filter 1,$V),,@)
```

Comment:
`$V` is uninitialized. Maybe it comes from [this Makefile](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/master/Makefile#L53). I don't know if it can cause problems, as I didn't well understand what that wrapper actually does.

Reply:
Removing it.
Thanks for the review